CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- [#12] Update Makefiles to 9.5.0
```

Comment:
Here is a heading missing that indicates the type of change. Keepachangelog.org has a handy list available.

Anyhow, I'll fix this during the release process.